build.sbt

```diff
@@ -1,61 +1,74 @@
 import sbt.Keys._
 import ReleaseTransformations._
 import Dependencies._
+import sbtversionpolicy.withsbtrelease.ReleaseVersion
 import sbtrelease.{Version, versionFormatError}
 
 /* --------------------------------------------------------------------- */
 
-val snapshotReleaseType = "snapshot"
-val snapshotReleaseSuffix = "-SNAPSHOT"
+//val snapshotReleaseType = "snapshot"
+//val snapshotReleaseSuffix = "-SNAPSHOT"
 
-lazy val versionSettingsMaybe = {
-  sys.props.get("RELEASE_TYPE").map {
-    case v if v == snapshotReleaseType => snapshotReleaseSuffix
-    case _ => ""
-  }.map { suffix =>
-    releaseVersion := {
-      ver => Version(ver).map(_.withoutQualifier.string).map(_.concat(suffix)).getOrElse(versionFormatError(ver))
-    }
-  }.toSeq
-}
+//lazy val versionSettingsMaybe = {
+//  sys.props.get("RELEASE_TYPE").map {
+//    case v if v == snapshotReleaseType => snapshotReleaseSuffix
+//    case _ => ""
+//  }.map { suffix =>
+//    releaseVersion := {
+//      ver => Version(ver).map(_.withoutQualifier.string).map(_.concat(suffix)).getOrElse(versionFormatError(ver))
+//    }
+//  }.toSeq
+//}
 
 lazy val root = (project in file("."))
   .aggregate(client, defaultClient)
-  .settings(commonSettings, versionSettingsMaybe, publishSettings)
+  .settings(commonSettings)
   .settings(
-    publish / skip     := true,
-    releaseVersionFile := baseDirectory.value / "version.sbt",
-    Compile / sources  := Seq.empty,
-    Test / sources     := Seq.empty
+    publish / skip := true,
+    //releaseVersionFile := baseDirectory.value / "version.sbt",
+    Compile / sources := Seq.empty,
+    Test / sources := Seq.empty,
+    releaseVersion := ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease().value,
+    releaseProcess := Seq(
+      checkSnapshotDependencies,
+      inquireVersions,
+      runClean,
+      runTest,
+      setReleaseVersion,
+      commitReleaseVersion,
+      tagRelease,
+      setNextVersion,
+      commitNextVersion
+    )
   )
 
 lazy val client = (project in file("client"))
-  .settings(commonSettings, clientSettings, publishSettings)
+  .settings(commonSettings, clientSettings)
   .enablePlugins(BuildInfoPlugin)
 
 lazy val defaultClient = (project in file("client-default"))
   .dependsOn(client)
-  .settings(commonSettings, defaultClientSettings, publishSettings)
+  .settings(commonSettings, defaultClientSettings)
 
 // we apply versionSettingsMaybe to aws too because this project is always released in isolation
 // from the others - and we _might_ want to put out a snapshot or beta for that (TBC).
 // Bear this in mind if we begin releasing aws in sync with the others
```

Comment:
Note that we should delete this comment too, as part of this PR, because it talks about `versionSettingsMaybe`, and we're deleting that on the line below.

```suggestion
// we apply versionSettingsMaybe to aws too because this project is always released in isolation
// from the others - and we _might_ want to put out a snapshot or beta for that (TBC).
// Bear this in mind if we begin releasing aws in sync with the others
```

The comment was added with PR https://github.com/guardian/content-api-scala-client/pull/324, and it makes an interesting point - that there essentially two separate groups of libraries in this repository, and they don't actually depend on each other - `content-api-client-aws` doesn't depend on `content-api-client`, `content-api-client-default`, etc, and they don't depend on it. They have different version numbers (`content-api-client-aws` is only at [version 0.7](https://search.maven.org/artifact/com.gu/content-api-client-aws_2.13)), and I think this leaves us with two courses of action:

* Synchronise them up and release them together, sharing a version number
* Separate `content-api-client-aws` out into its own GitHUb repository, and have a separate release setup for that - right now, I think this is the one I would prefer.